operator/Makefile

```diff
@@ -33,25 +40,32 @@ IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 
-# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
-ifeq (,$(shell go env GOBIN))
-GOBIN=$(shell go env GOPATH)/bin
-else
-GOBIN=$(shell go env GOBIN)
-endif
+K8S_VER=1.19.x
+
+TARGET = ${MKFILE_DIR}bin/manager
+ALL_SOURCE_FILES = $(shell find ${MKFILE_DIR} -type f -name "*.go")
 
-all: manager
 
+all: manager
+#export KUBEBUILDER_ASSETS=/Users/chenhao/GitHub/easemesh/operator/testbin/bin;
 # Run tests
-ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
-test: generate fmt vet manifests
+ENVTEST_ASSETS_DIR=${MKFILE_DIR}/testbin
+test1: generate fmt vet manifests setup_envtest
```

Comment:
Propose a formal target name, such as `test_k8s_version`

Reply:
this just uses the new tool. how about change to `newtest`?

Reply:
Good to me